### PR TITLE
New feature: inverted buzzer

### PR DESCRIPTION
**Pull request by ep092**
I added support for hardware from https://github.com/Virtex7/RX5808
This board has a different wiring at the buzzer so the buzzer output needs to be inverted.

### CODE REVIEW

File: src/rx5808-pro-diversity/rx5808-pro-diversity.ino [hunk 1]
```diff
@@ -63,7 +63,7 @@ const uint16_t channelTable[] PROGMEM = {
   0x2903,    0x290C,    0x2916,    0x291F,    0x2989,    0x2992,    0x299C,    0x2A05,    // Band B
   0x2895,    0x288B,    0x2881,    0x2817,    0x2A0F,    0x2A19,    0x2A83,    0x2A8D,    // Band E
   0x2906,    0x2910,    0x291A,    0x2984,    0x298E,    0x2998,    0x2A02,    0x2A0C,    // Band F / Airwave
-  0x281D,    0x288F,    0x2902,    0x2914,    0x2978,    0x2999,    0x2A0C,    0x2A1E     // Band C / Immersion Raceband
+  0x281D,    0x288F,    0x2902,    0x2914,    0x2987,    0x2999,    0x2A0C,    0x2A1E     // Band C / Immersion Raceband
```
Comment (sheaivey) [inline src/rx5808-pro-diversity/rx5808-pro-diversity.ino]: Glad you got this fix in.

File: src/rx5808-pro-diversity/settings.h [hunk 2]
```diff
@@ -85,6 +84,13 @@ SOFTWARE.
 // Buzzer
 #define buzzer 6
 
+//activate if your buzzer is inverted compared to normal layout. 
+//#define BUZZER_INVERTED
+
+//aktivate to switch on a externel display with a short impulse on rxd
+//#define ACTIVATE_EXTERNAL_SCREEN
+#define RXD 0
```
Comment (sheaivey) [inline src/rx5808-pro-diversity/settings.h]: Everything is fine with your PR except i do not know what these lines are for.
90, 91, 92

If you remove them I'll merge the PR thanks.

**Comment by ep092** [comment]
This commits were after my pullrequest. I did not know that also future commits will be part of the pullrequest. So this lines do not need to be mergerd. 

The reason for line 90,91,92 are, that i use a laptop screen with an lvds interface. This lvds interface is switched on with Pin 0 of the atmel after some time since start.

**Comment by sheaivey** [comment]
No worries @ep092 ... Unfortunately this PR is a little old now and has conflicts. I'm going to close it as some one else has submitted the fix for fixing channel C5.

**Pull request closed without merging**